frontend/public/components/factory/list.tsx

```diff
@@ -290,7 +290,7 @@ export class ColHead extends React.Component<ColHeadProps> {
       sortField,
       sortFunc,
     } = this.props;
-    const className = classNames(this.props.className, 'text-nowrap');
+    const className = classNames(this.props.className);
```

Comment:
You should not longer need to use `classNames`. Just use `this.props.className` below. (I'd add that to the destructured props just above.)

Reply:
duh :-) sorry, fixed